server.js

```diff
@@ -16,7 +16,7 @@ const app = express();
 app.use(helmet({
     contentSecurityPolicy: {
         directives: {
-            defaultSrc: ["'none'"],
+            defaultSrc: ['\'none\''],
```

Comment:
If you find this disagreeable, it looks ESLint's `quotes` rule has a special case for [`avoidEscape`](https://eslint.org/docs/rules/quotes#avoidescape) we could employ here.

Reply:
Ah. Yeah, I'd be down for using `avoidEscape`.